tcms/testruns/tests/test_admin.py

```diff
@@ -22,3 +29,23 @@ def test_regular_user_can_not_change_testrun(self):
         response = self.client.get(reverse('admin:testruns_testrun_change',
                                            args=[self.test_run.pk]))
         self.assertRedirects(response, reverse('testruns-get', args=[self.test_run.pk]))
+
+    def test_regular_user_can_not_delete_testrun(self):
+        response = self.client.get(reverse('admin:testruns_testrun_delete',
+                                           args=[self.test_run.pk]), follow=True)
+        self.assertContains(response, _('Permission denied: TestRun does not belong to you'))
+
+    def test_superuser_can_delete_testrun(self):
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.superuser.username,
+            password='password')
+        response = self.client.get(reverse('admin:testruns_testrun_delete',
+                                           args=[self.test_run.pk]))
+        self.assertContains(response, _("Yes, I'm sure"))
+        response = self.client.post(reverse('admin:testruns_testrun_delete',
+                                            args=[self.test_run.pk]), {'post': 'yes'}, follow=True)
+        self.assertRedirects(response, reverse('admin:testruns_testrun_changelist'))
+
+    def test_superuser_can_delete_testrun_verify(self):
+        trpk = self.test_run.pk
+        return TestRun.objects.filter(pk=trpk).exists()
```

Comment:
These 2 lines can be condensed into 1 and need to be at the end of test_superuser_can_delete_testrun method, not separate. 

Also `return` here is not needed.